.pipelines/e2e-job-azure.yaml

```diff
@@ -20,5 +20,6 @@ jobs:
       - "linux"
       - "windows_docker"
       - "windows_containerd"
+  # ToDo: re-enable this job after implementing automated ext release process
```

Comment:
```suggestion
  # TODO: re-enable this job after implementing automated ext release process
  # using https://github.com/Azure/secrets-store-csi-driver-provider-azure/issues/1382 for tracking
```

Reply:
Should we reopen https://github.com/Azure/secrets-store-csi-driver-provider-azure/issues/1382?

Reply:
done